src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+
+    public static Figure getRandomFigure() {
+        int choice = random.nextInt(5);
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String randomColor = colorSupplier.getRandomColor();
+
+        switch (choice) {
+            case 0:
+                return new Square(random.nextInt(10) + 1, randomColor);
+            case 1:
+                return new Rectangle(random.nextInt(10) + 1, random.nextInt(10) + 1, randomColor);
+            case 2:
+                return new RightTriangle(random.nextInt(10) + 1,
+                        random.nextInt(10) + 1, randomColor);
+            case 3:
+                return new Circle(random.nextInt(10) + 1, randomColor);
+            default:
```

Comment:
```suggestion
            case 4:
            default:
```
better to clarify 4th option

Reply:
not fixed